src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly

```diff
@@ -0,0 +1,65 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="URL" field="url">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="Name" field="name">
+        <f:textbox />
+    </f:entry>
+
+    <f:advanced>
+        <f:section title="Build status">
+            <f:description>
+                This section defines for which build statuses the notification is sent.
+            </f:description>
+
+            <f:entry title="Notify Build Start" field="startNotification">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Aborted" field="notifyAborted">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Failure" field="notifyFailure">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Not Built" field="notifyNotBuilt">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Success" field="notifySuccess">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Unstable" field="notifyUnstable">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Back To Normal" field="notifyBackToNormal">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Repeated Failure" field="notifyRepeatedFailure">
+                <f:checkbox />
+            </f:entry>
+        </f:section>
+
+        <f:section title="Advanced configuration">
+            <f:entry title="Macros" field="macros">
+                <f:repeatableProperty minimum="0" field="macros" add="${%Add Macro}">
+                </f:repeatableProperty>
+            </f:entry>
+            <f:entry title="Timeout" description="Timeout (in ms)" field="timeout">
+                <f:number default="${descriptor.defaultTimeout}"/>
+            </f:entry>
+        </f:section>
+
+    </f:advanced>
+    <f:entry title="">
```

Comment:
is ``title`` attribute needed?

Reply:
I don't think so, must have started typing and forgot to delete 👍